homeassistant/components/homematic/entity.py

```diff
@@ -27,20 +30,27 @@
 class HMDevice(Entity):
     """The HomeMatic device base object."""
 
-    def __init__(self, config):
+    def __init__(
+        self,
+        config: dict[str, str],
+        entity_description: SensorEntityDescription | None = None,
+    ) -> None:
         """Initialize a generic HomeMatic device."""
         self._name = config.get(ATTR_NAME)
         self._address = config.get(ATTR_ADDRESS)
         self._interface = config.get(ATTR_INTERFACE)
         self._channel = config.get(ATTR_CHANNEL)
         self._state = config.get(ATTR_PARAM)
         self._unique_id = config.get(ATTR_UNIQUE_ID)
-        self._data = {}
+        self._data: dict[str, str] = {}
```

Comment:
I had to set a type for this, because mypy gave me an error.  I think this one should be a dict[str, str] or am I wrong?